docs/ref/contrib/gis/model-api.txt

```diff
@@ -240,12 +241,20 @@ For more information, the PostGIS documentation contains a helpful section on
 determining `when to use geography data type over geometry data type
 <http://postgis.net/docs/manual-2.1/using_postgis_dbmanagement.html#PostGIS_GeographyVSGeometry>`_.
 
-``GeoManager``
-==============
+``GeoManager`` WARNING: DEPRECATED
+===================================
 
 .. currentmodule:: django.contrib.gis.db.models
 .. class:: GeoManager
 
+.. note::
```

Comment:
use `.. deprecated:: 1.9` instead of note and include an explanation of why the class is deprecated.